### PR TITLE
Add missing `encodeURIComponent()` around `text` and `url`

### PR DESCRIPTION
**Pull request by valentinoli**
I tested the polyfill on text that contained hashtags and in many cases the link would not work because the text not been encoded.

### CODE REVIEW

**Comment by valentinoli** [comment]
Addresses #76

File: src/share.js [hunk 1]
```diff
@@ -530,7 +530,7 @@ navigator.share = navigator.share || (function () {
 			function bindEvents() {
 				Array.from(container.querySelectorAll('.tool-icon')).forEach(tool => {
 					tool.addEventListener('click', event => {
-						const payload = text + ': ' + url;
+						const payload = encodeURIComponent(text + ': ' + url);
```
Comment (valentinoli) [inline src/share.js:533]: not sure this about this one

**Comment by JohMun** [comment]
I have the same issue in current firefox. If you pass a URL like `https://domain?test=3&test2=3`, parts of the URL are missing, because this library doesn't encode it. It would be nice if a new version is published

**Comment by valentinoli** [comment]
Hey @felipenmoura, I'm wondering if you would have any time soon to take a look at this and the other PRs I submitted.

**Comment by felipenmoura** [comment]
Hey @valentinoli thanks for the PR.
Sorry it took so long for being merged.

**Pull request merged**